sibyl/resources/computing.py

```diff
@@ -15,6 +14,29 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def validate_changes(changes):
+    """
+    Helper function for validating changes to entity.
+    """
+    for change in changes:
+        change[0] = str(change[0])
+        try:
+            change[1] = float(change[1])
+        except:
```

Comment:
To avoid the unspecified exception code-smell here (I think its pre-existing, but worth fixing while you're in the code), please add the type of error (I think you want `except ValueError`)

Reply:
I think this code intends to force the change into a float number if it is a number and do nothing if it is not. I can change this into an if statement. Sometimes the feature value is a string instead of a number.

This does bring up another issue. The validation here does not necessarily check if the type of the change value is always the same as the type of the original feature value. I will make an issue, but this should not affect my purpose too much.